corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -168,3 +169,75 @@ def send_HTML_email(subject, recipient, html_content, text_content=None,
                 bcc=bcc,
             )
             error_msg.send()
+
+
+def getEmailConfiguration(domain: str, from_email: str = settings.DEFAULT_FROM_EMAIL):
```

Comment:
why the camelCase?

Reply:
oops [a0a399c](https://github.com/dimagi/commcare-hq/pull/33484/commits/a0a399cbcdab3671bff35843a360912c82d1b7c1)